### PR TITLE
test(skiplink-list): add visual spec

### PR DESCRIPTION
**Pull request by TomMenga**

### CODE REVIEW

File: src/elements/skiplink-list/skiplink-list.visual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+`;
+
+describe(`sbb-skiplink-list`, () => {
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    it(
+      'not focused',
+      visualDiffDefault.with(async (setup) => {
+        await setup.withFixture(
+          html` <sbb-skiplink-list title-content="Skip to"> ${links()} </sbb-skiplink-list> `,
+          { minHeight: '100px' },
```
Comment (DavideMininni-Fincons) [inline src/elements/skiplink-list/skiplink-list.visual.spec.ts:21]: you can refactor this by creating a const and passing it in tests

File: src/elements/skiplink-list/skiplink-list.visual.spec.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+const links = (): TemplateResult => html`
+  <sbb-block-link href="#">Link 1</sbb-block-link>
+  <sbb-block-link href="#">Link 2</sbb-block-link>
+  <sbb-block-link href="#">Link 3</sbb-block-link>
+`;
+
+describe(`sbb-skiplink-list`, () => {
+  describeViewports({ viewports: ['zero', 'medium'] }, () => {
+    it(
+      'not focused',
```
Comment (DavideMininni-Fincons) [inline src/elements/skiplink-list/skiplink-list.visual.spec.ts:17]: this looks like the 'no title' but unfocused; could we have a for cycle with default/focus?

**Review by DavideMininni-Fincons — COMMENTED** [review]
Looks good, some comments added

File: src/elements/skiplink-list/skiplink-list.visual.spec.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            <sbb-skiplink-list>
+              <span slot="title">Slotted title</span>
+              ${links()}
+            </sbb-skiplink-list>
+          `,
+          { minHeight: '100px' },
+        );
+      }),
+    );
+  });
```
Comment (jeripeierSBB) [inline src/elements/skiplink-list/skiplink-list.visual.spec.ts:63]: Should we really test visually hidden stuff? btw. there is a visual-level attribute on the title which could be removed

Comment (TomMenga) [reply src/elements/skiplink-list/skiplink-list.visual.spec.ts:63]: Maybe I'm being too zealous but I believe it's the only way we have to be sure that they stay hidden :/

Comment (jeripeierSBB) [reply src/elements/skiplink-list/skiplink-list.visual.spec.ts:63]: ok that's true, but maybe remove just the slottted one?

**Review by DavideMininni-Fincons — APPROVED** [review]
LGTM

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**